txscript/standard_test.go

```diff
@@ -1013,3 +1019,89 @@ func TestStringifyClass(t *testing.T) {
 		}
 	}
 }
+
+// TestNullDataScript tests whether NullDataScript returns a valid script.
+func TestNullDataScript(t *testing.T) {
+	tiny, _ := hex.DecodeString("01")
+	short, _ := hex.DecodeString("0102030405060708090a0b0c0d0e0f1" +
+		"01112131415161718")
+	shortExp, _ := hex.DecodeString("6a180102030405060708090a0b0c0d0e0f1" +
+		"01112131415161718")
+	justRight, _ := hex.DecodeString("000102030405060708090a0b0c0d0e0f" +
+		"101112131415161718191a1b1c1d1e1f20212223242526272829" +
+		"2a2b2c2d2e2f303132333435363738393a3b3c3d3e3f40414243" +
+		"4445464748494a4b4c4d4e4f")
+	tooLong, _ := hex.DecodeString("000102030405060708090a0b0c0d0e0f" +
+		"101112131415161718191a1b1c1d1e1f20212223242526272829" +
+		"2a2b2c2d2e2f303132333435363738393a3b3c3d3e3f40414243" +
+		"4445464748494a4b4c4d4e4f50")
+
+	tests := []struct {
+		name     string
+		data     []byte
+		valid    bool
+		expected []byte
+	}{
+		{
+			name:     "small data",
+			data:     tiny,
```

Comment:
Ok, I see it's `decodeHex` in the test package.  I'll need to combine those when doing aforementioned PR.